tools/plugins/com.liferay.ide.lsp.ui/grammars/ini/ini.tmLanguage.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
Is this the grammer for a 'properties' file?  What is it called `ini`?  Is that what it is based off of?  Lets rename this file.